packages/js-cloudbees-provider/src/lib/js-cloudbees-provider.ts

```diff
@@ -1,17 +1,17 @@
+import { parseValidJsonObject } from '@openfeature/extra';
 import {
-  FeatureProvider,
-  parseValidJsonObject,
-  ProviderEvaluation,
+  Provider,
+  EvaluationContext,
+  ResolutionDetails,
   ProviderOptions,
-  Reason,
 } from '@openfeature/openfeature-js';
 import * as Rox from 'rox-node';
 
 export interface CloudbeesProviderOptions extends ProviderOptions {
   appKey: string;
 }
 
-export class CloudbeesProvider implements FeatureProvider {
+export class CloudbeesProvider implements Provider {
```

Comment:
I think we should add comments to all the providers saying they are unofficial, and basically PoCs.